base_layer/core/src/base_node/states/listening.rs

```diff
@@ -64,6 +64,7 @@ impl ListeningData {
     pub async fn next_event<B: BlockchainBackend>(&mut self, shared: &mut BaseNodeStateMachine<B>) -> StateEvent {
         info!(target: LOG_TARGET, "Listening for chain metadata updates");
         shared.info = StatusInfo::Listening(ListeningInfo::new());
+        let _ = shared.status_event_publisher.send(shared.info.clone()).await;
```

Comment:
Ideally, publishing an event would just be wrapped in a function
```suggestion
        shared.publish_status_event(shared.info.clone()).await;
```